app/src/main/java/org/apache/taverna/mobile/ui/anouncements/AnnouncementPresenter.java

```diff
@@ -26,23 +26,21 @@
 import org.apache.taverna.mobile.data.model.DetailAnnouncement;
 import org.apache.taverna.mobile.ui.base.BasePresenter;
 
-import rx.Observer;
-import rx.android.schedulers.AndroidSchedulers;
-import rx.schedulers.Schedulers;
-import rx.subscriptions.CompositeSubscription;
-
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.observers.DisposableObserver;
+import io.reactivex.schedulers.Schedulers;
+import io.reactivex.disposables.CompositeDisposable;
 
 public class AnnouncementPresenter extends BasePresenter<AnnouncementMvpView> {
 
     public final String LOG_TAG = getClass().getSimpleName();
-    private DataManager mDataManager;
-    private CompositeSubscription mSubscriptions;
 
+    private DataManager mDataManager;
+    private CompositeDisposable compositeDisposable;
 
     public AnnouncementPresenter(DataManager dataManager) {
         mDataManager = dataManager;
-
-        mSubscriptions = new CompositeSubscription();
+        compositeDisposable = new CompositeDisposable();
```

Comment:
Good work for using the `CompositeDisposable`, It has so many advantages that why we exceptionally lose subscription because of our logic.